docs/tutorials/getting-started.rst

```diff
@@ -0,0 +1,125 @@
+Getting Started
+===============
+
+This tutorial will guide you towards a working Taskcluster + Taskgraph setup
+with your repo in as few steps as possible.
+
+.. note::
+
+   This tutorial has a heavy focus on the end result. If you prefer to build
+   understanding as you go, you may wish to follow the
+   :doc:`creating-a-task-graph` and :doc:`connecting-taskcluster` tutorials
+   instead.
+
+There are three steps to getting running tasks:
+
+1. Install Taskcluster Github
+2. Request Permissions
+3. Define Tasks
+
+Let's dive into each!
+
+Install Taskcluster Github
+--------------------------
+
+.. note::
+
+   If you are not using Github, skip to the next section.
+
+Each Taskcluster deployment has an associated Github app that needs to be added
+to your repo. Some apps for known Taskcluster deployments are as follows:
+
+.. list-table:: Taskcluster Github Apps
+   :widths: 20 60
+   :header-rows: 1
+
+   * - Instance
+     - Github App
+   * - `Firefox-CI <https://firefox-ci-tc.services.mozilla.com>`_
+     - https://github.com/apps/firefoxci-taskcluster
```

Comment:
Thanks for documenting these URLs! I struggled so hard to find them, I had to bookrmark them 😅